exchanges/bybit/bybit_websocket.go

```diff
@@ -793,3 +671,172 @@ const subTplText = `
+	var args []SubscriptionArgument
+	arg := SubscriptionArgument{
+		Operation: operation,
+		RequestID: strconv.FormatInt(conn.GenerateMessageID(false), 10),
+		Arguments: []string{},
+	}
+	authArg := SubscriptionArgument{
+		auth:      true,
+		Operation: operation,
+		RequestID: strconv.FormatInt(conn.GenerateMessageID(false), 10),
+		Arguments: []string{},
+	}
+
+	chanMap := map[string]bool{}
+	pairFmt, err := e.GetPairFormat(assetType, true)
+	if err != nil {
+		return nil, err
+	}
+	for _, s := range channelsToSubscribe {
+		var pair currency.Pair
+		if len(s.Pairs) > 1 {
+			return nil, subscription.ErrNotSinglePair
+		}
+		if len(s.Pairs) == 1 {
+			pair = s.Pairs[0]
+		}
+		switch s.Channel {
+		case chanOrderbook:
+			arg.Arguments = append(arg.Arguments, fmt.Sprintf("%s.%d.%s", s.Channel, 50, pairFmt.Format(pair)))
+			arg.associatedSubs = append(arg.associatedSubs, s)
+		case chanPublicTrade, chanPublicTicker, chanLiquidation, chanLeverageTokenTicker, chanLeverageTokenNav:
+			arg.Arguments = append(arg.Arguments, s.Channel+"."+pairFmt.Format(pair))
+			arg.associatedSubs = append(arg.associatedSubs, s)
+		case chanKline, chanLeverageTokenKline:
+			interval, err := intervalToString(kline.FiveMin)
+			if err != nil {
+				return nil, err
+			}
+			arg.Arguments = append(arg.Arguments, s.Channel+"."+interval+"."+pairFmt.Format(pair))
+			arg.associatedSubs = append(arg.associatedSubs, s)
+		case chanPositions, chanExecution, chanOrder, chanWallet, chanGreeks:
+			if chanMap[s.Channel] {
+				continue
+			}
+			authArg.Arguments = append(authArg.Arguments, s.Channel)
+			// add channel name to map so we only subscribe to channel once
+			chanMap[s.Channel] = true
+			authArg.associatedSubs = append(authArg.associatedSubs, s)
+		}
+
+		if len(arg.Arguments) >= 10 {
+			args = append(args, arg)
+			arg = SubscriptionArgument{
+				Operation: operation,
+				RequestID: strconv.FormatInt(conn.GenerateMessageID(false), 10),
+				Arguments: []string{},
+			}
+		}
+	}
+	if len(arg.Arguments) != 0 {
+		args = append(args, arg)
+	}
+	if len(authArg.Arguments) != 0 {
+		args = append(args, authArg)
+	}
+	return args, nil
+}
+
+// generateAuthSubscriptions generates default subscription for the dedicated auth websocket connection. These are
+// agnostic to the asset type and pair as all account level data will be routed through this connection.
+// TODO: Remove this function when template expansion is across all assets
+func (e *Exchange) generateAuthSubscriptions() (subscription.List, error) {
+	if !e.Websocket.CanUseAuthenticatedEndpoints() {
+		return nil, nil
+	}
+
+	for _, configSub := range e.Config.Features.Subscriptions.Enabled() {
+		if configSub.Authenticated {
+			log.Warnf(log.WebsocketMgr, "%q has an authenticated subscription %q in config which is not supported. Please remove", e.Name, configSub.Channel)
```

Comment:
```suggestion
			log.Warnf(log.WebsocketMgr, "%s has an authenticated subscription %q in config which is not supported. Please remove.", e.Name, configSub.Channel)
```

Just a consistency nit since we generally use the exchange name without quotes